eng/pipelines/templates/steps/analyze.yml

```diff
@@ -28,6 +28,7 @@ steps:
       SourceDirectory: $(Build.SourcesDirectory)
 
   - template: /eng/common/pipelines/templates/steps/verify-links.yml
+    condition: ne(variables['Skip.LinkCheck'], 'true')
```

Comment:
When/where is this variable set?

Reply:
Queue time variable

Reply:
Please hold on. YAML has formatting error. I will push new change.

Reply:
Are users expected to set the queue time variable manually?  I'm still not sure how this works end-to-end.

Reply:
They are not expected to set if they don't want to skip link check. This needs to be set only if anyone wants to skip link check on this hotfix branch.

Reply:
Should link check be skipped in all hotfix branches?  If so, can we detect this automatically without requiring a queue-time variable?

Reply:
We will have to patch all those branches in that case. right? Queue time variable was a quick work around to unblock release.
Ideal solution is to compare within branch if build is not triggered from main branch.